Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM alpine:3.12
+FROM alpine:3.12.1
```

Comment:
Is there a particular reason for this change?
For information, the Docker Image is rebuilt automatically when the upstream `alpine:3.12` image is updated,
so this change is not needed:

<img width="1232" alt="Screenshot 2020-12-10 at 18 31 14" src="https://user-images.githubusercontent.com/1522731/101808001-130efa00-3b16-11eb-87a0-92bd6c0bd72e.png">

Reply:
```suggestion
FROM alpine:3.12
```